Makefile

```diff
@@ -5,12 +5,16 @@ AWS_DISTRIBUTE=
 BUILDDIR=.build
 MAINTAINER_EMAIL="contact@gardenlinux.io"
 
+ifdef LOCAL_PKGS
+override BUILD_OPTS += --local-pkgs=$(LOCAL_PKGS)
```

Comment:
I'm sceptical about this "get a directory from an environment variable". Wouldn't it be more useful if there would exist an directory in the build tree where a proper README can reside and where you just need to put the file?